client/quantum_serverless/core/provider.py

```diff
@@ -321,14 +321,6 @@ def __init__(
             available_compute_resources: available clusters in provider
         """
         super().__init__(name)
-        warnings.warn(
-            "`KuberayProvider` is deprecated "
-            "and will be removed in v0.3. "
```

Comment:
I don't think we should drop this before v0.3, probably better to have pylint ignore it via something like `# pylint: disable=undefined-variable`

Reply:
@psschwei thanks for review.  It looks like this should have forgotten to  take out as a part of #697.

Reply:
good catch -- yes, we should add that import back

Reply:
OK.  I'll do that.  Thanks.